models/tmp/stg_zendesk__ticket_schedule_tmp.sql

```diff
@@ -0,0 +1,23 @@
+{%- set source_relation = adapter.get_relation(
+      database=source('zendesk', 'ticket_schedule').database,
```

Comment:
what happens here if the customer's default schema name is not `zendesk`? perhaps it's `zendesk_fivetran` or something like that.  Will it still work?

Reply:
That's a good question and one I did not think of. Thankfully, I just tested this on a dataset with a schema different than `zendesk` (it also had a `ticket_schedules` table populated) and the different schema name worked and the `stg_zendesk__ticket_schedule` table was correctly populated with values rather than the one row of nulls!

I am happy to walk you through this test if you would like! It seems setting the `zendesk_schema` variable ensures this will work.